clients/spark/project/plugins.sbt

```diff
@@ -4,3 +4,4 @@ addSbtPlugin("com.frugalmechanic" % "fm-sbt-s3-resolver" % "0.19.0")
 addSbtPlugin("cf.janga" % "sbts3" % "0.10.5")
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.29")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.0")
+addSbtPlugin("com.github.sbt" % "sbt-jacoco" % "3.3.0")
```

Comment:
Why is this in the project plugins?  AFAICT it is never used.  Perhaps put it in your personal plugins in `~/.sbt/1.0/plugins/plugins.sbt`?

Reply:
I disagree.  It is not necessary to build or to test.  I searched for "sbt coverage" and I discovered [sbt-coverage](https://github.com/scoverage/sbt-scoverage).  Now we can have an argument which coverage plugin should we use?  E.g. think of participating in Hacktoberfest with such a line.

Basically I do not believe that we care to recommend a coverage plugin.  I regard this as part of preferred tooling.  We should avoid being opinionated here because:
a. We _can_; and
b. it's not our part of our core value.